bats/tests/k8s/enable-disable-k8s.bats

```diff
@@ -0,0 +1,39 @@
+# Test case 8 & 13
+
+setup() {
+    load '../helpers/load'
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+verify_k8s_is_running() {
+    wait_for_container_engine
+    run kubectl cluster-info
+    assert_output --regexp "Kubernetes control plane.*is running"
+    assert_output --regexp "CoreDNS.*is running"
+    assert_output --regexp "Metrics-server.*is running"
+    # regexp is needed to skip the color control codes with `.*`
+}
+
+@test 'start rancher desktop with kubernetes enabled' {
+    start_kubernetes
+    wait_for_apiserver
+    verify_k8s_is_running
+}
+
+@test 'disable kubernetes' {
+    rdctl set --kubernetes-enabled=false
+    wait_for_container_engine
+    run kubectl cluster-info
```

Comment:
I'm wondering how we can really prove that k8s is not going to start up again. Here you are running `kubectl cluster-info` as soon as the container engine is ready. But at that time the apiserver is not going to be ready anyways, so this test would probably pass even if Kubernetes was still enabled.

I don't have a good suggestion on how you can prove that Kubernetes won't start again "soon".

The best I came up with is checking the service state, e.g.

```sh
run rdsudo "$RC_SERVICE" --nocolor k3s status
# rc-service seems to use exit code 3 when the service is not running
assert_failure
assert_line --partial "status: stopped"
```

Reply:
Done. I also added a 60-second wait after the container engine is running before testing for k8s.

And I'm using the same technique to verify it is running -- just ask `rc-service`